default/python/AI/FreeOrionAI.py

```diff
@@ -96,8 +98,25 @@ def _pre_game_start(empire_id, aistate):
     configure_debug_chat(empire_id)
 
 
+def _choose_aggression():
+    galaxy_setup_data = fo.getGalaxySetupData()
+    aggression = int(galaxy_setup_data.maxAIAggression)
+
+    if aggression > 0:
+        rng = random.Random()
+        galaxy_seed = hash(galaxy_setup_data.seed)
+        empire_seed = 3 * hash(fo.getEmpire().name)
+        rng.seed(galaxy_seed * empire_seed)
+
+        random_number = rng.randint(0, 99)
```

Comment:
https://docs.python.org/3/library/random.html#random.choices

```python
aggression _modifier = choices((0, -1), cum_weights=(74, 100))[0]
```

Reply:
While I like to use this function a lot when there are multiple choices or multiple samples involved, I find it adds quite a bit of cognitive overhead in this simple case and prefer the if-statement.

If you have a strong preference, I am of course willing to change it.